doc/releasenotes/16_0_0_summary.md

```diff
@@ -144,6 +149,28 @@ The `RestoreFromBackup  --restore_to_pos` ends with:
 - the restored server in intentionally broken replication setup
 - tablet type is `DRAINED`
 
+### Error codes
+
+In this release, we are introducing a new way to handle errors from Vitess through the query interface. 
```

Comment:
```suggestion
In this release, we are introducing a new way to report errors from Vitess through the query interface. 
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)